spark-bigtable/spark-bigtable-scala2.13/pom.xml

```diff
@@ -0,0 +1,149 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>com.google.cloud.spark.bigtable</groupId>
+    <artifactId>spark-bigtable</artifactId>
+    <version>0.3.0</version> <!-- ${NEXT_VERSION_FLAG} -->
+  </parent>
+
+  <artifactId>spark-bigtable-scala2.13</artifactId>
+  <version>0.3.0</version> <!-- ${NEXT_VERSION_FLAG} -->
+
+  <properties>
+    <scala.version>2.13.14</scala.version>
+    <scala.binary.version>2.13</scala.binary.version>
+    <spark.version>3.5.1</spark.version>
+  </properties>
+
+  <dependencies>
```

Comment:
similar to the other pom file, can we add comments on the section of the dependencies? scala dependencies, spark depedencens, test dependencies etc?